tests/e2e/specs/create_and_remove-existing-menu-test.js

```diff
@@ -0,0 +1,59 @@
+import { loginUser, visitAdminPage,activateTheme } from '@wordpress/e2e-test-utils';
+
+describe( 'Should create a new menu and remove existing menu', () => {
+
+    beforeEach( async () => {
+        await loginUser();
+	await activateTheme("twentytwentyone");
+    } );
+
+
+	it( 'Add a new menu', async () => {
+		
+        await visitAdminPage("nav-menus.php");
+        
+        //check if it's a first menu
+        await page.waitForSelector(".first-menu-message")
```

Comment:
Can you clarify what this line does exactly? It looks like you are waiting for a selector that does not exist?